src/components/atoms/UserStatusDropdown/UserStatusDropdown.tsx

```diff
@@ -1,39 +1,47 @@
-import React, { useMemo } from "react";
+import React, { useMemo, useEffect } from "react";
 import { Dropdown, DropdownButton } from "react-bootstrap";
 
-import { USER_STATUSES } from "settings";
+import { UserStatus } from "types/User";
 
 import { useProfileStatus } from "hooks/useProfileStatus";
 
-import { UserStatus } from "types/User";
-
 import "./UserStatusDropdown.scss";
 
 export interface UserStatusDropdownProps {
   userStatuses?: UserStatus[];
 }
 
 export const UserStatusDropdown: React.FC<UserStatusDropdownProps> = ({
-  userStatuses,
+  userStatuses = [],
 }) => {
   const { status, changeUserStatus } = useProfileStatus();
```

Comment:
can we use `useVenueUserStatuses` hook here for `status`? looks like it knows what to do when there is no existent status. (and no need of the `useEffect` then). What do you think? 🤔

Reply:
Actually, it makes sense. I will see if I can replace it with the other one and remove `useProfileStatus` because this is the only place it's being used.

Reply:
The result is the same :/ Both hooks only get the status from the database, but if that status was set 5-10 days ago and the statuses have changed, the hook is still required and needs to check if that status exists.

Reply:
I can move `changeUserStatus` in `useVenueUserStatuses` hook and remove `useProfileStatus` but the result would be the same. It's your call. I don't mind refactoring it in here.

Reply:
@denisdimitrov I believe both hooks serve similar purposes. So I'd merge the functionality. Can we do that?

Reply:
@sunny-viktoryia I'll merge it and commit in about 10 minutes, are you going to be around to approve this?

Reply:
yes, I'll be here.

Reply:
Unless github decides that this is eng-lead worthy PR and doesn't accept your approve 😄

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1772/commits/80aec3855dd428a02639aa6009e990f5fc8e837e

Reply:
The user statuses were broken after that commit so I had to add some additional changes as well.